cms/djangoapps/contentstore/views/videos.py

```diff
@@ -328,6 +331,13 @@ def videos_post(course, request):
     for req_file in req_files:
         file_name = req_file["file_name"]
 
+        # Check if file_name contains non-ascii characters.
+        try:
+            file_name.encode('ascii')
+        except UnicodeEncodeError as exception:  # pylint: disable=unused-argument
+            error_msg = "%s contains non ascii characters in file name." % file_name
```

Comment:
@catong User can upload file name like `code+-~!@#$%^&*().mp4` which is permisable by amazon s3, so only mentioning alphanumeric would be good?

@sstack22 Do we only need to restrict to alphanumneric and underscores/period characters only ? or we should also allow other characters which are ascii just as amazon allow us.